packages/client-core/src/components/ViewerInteractions/index.tsx

```diff
@@ -50,8 +50,10 @@ export const ViewerInteractions = () => {
   const loadingScreenVisible = useHookstate(getMutableState(LoadingSystemState).loadingScreenVisible).value
   const { t } = useTranslation()
   const externalInjectedMenus = useMutableState(ViewerMenuState).externalInjectedMenus.get(NO_PROXY)
+  const locationContainer = useRef<HTMLDivElement>(null)
 
   useLayoutEffect(() => {
+    if (locationContainer.current) locationContainer.current.style.opacity = '0'
```

Comment:
this should be in it's own useLayoutEffect with locationContainer.current as the dep